### ISSUE TITLE
Fields mapping - broad usage

### ISSUE THREAD

**Issue by SubjectXXX** [Hafinator/LazyFURS#1]
Greetings,

Is there a way to implement fields remapping, so we could use different exports, like from Trading212 or simliar?

**Comment by Hafinator** [comment]
Hmmm that would be something completely different from what I originally intended this app to be.
It is doable, but if I expose this option in the menu, I worry that non-programmer-savvy users might get confused.

**Comment by SubjectXXX** [comment]
Well, confusion is avoided, if there is documentation or some instructions on particular functionality.

I dont see a problem with new menu item, that, once clicked, pops up warning, that this is meant for something something..

**Comment by Hafinator** [comment]
Yeah, you're probably right. I'll look into it, but not this tax season.
